python/data/chap3/README.rst

```diff
@@ -20,6 +20,33 @@ start.dlis.part              SUL + all the VR and LRSH bytesiBytes 50 and 51
 /reprcode                    Object attribute of various representation codes
 ============================ =================================================
 
+implicit folder contains full files and it's worth to describe it in more
+details.
+
+====================================== ========================================
+Files in /implicit                     Description
+====================================== ========================================
+fdata-broken-obname.dlis               File is truncated in the middle of obname
+                                       in object description in fdata
+
+fdata-encrypted.dlis                   Fdata record is encrypted
+
+fdata-many-in-same-vr.dlis             Several fdata segments in same vr,
```

Comment:
We probably want one with min_obname_size aswell (`idlen` is 0), if do not already have it?